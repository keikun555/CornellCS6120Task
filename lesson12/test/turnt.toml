[envs.original-eval]
command = "bril2json < {filename} | brili {args}"
output.out = "-"
output.err = "2"

[envs.copy-eval]
command = "bril2json < {filename} | deno run --no-config '../brili.ts' {args}"
output.out = "-"
output."copy.err" = "2"

[envs.trace]
command = "bril2json < {filename} | deno run --no-config '../brili.ts' -t {args}"
output."bril.trace.out" = "-"
output."bril.trace.err" = "2"

[envs.jit]
command = "bril2json < {filename} | python3 ../jit.py {filename}.trace.err | bril2txt"
output."jit.out" = "-"
output."jit.err" = "2"

[envs.jit-eval]
command = "bril2json < {filename} | python3 ../jit.py {filename}.trace.err | brili {args}"
output.out = "-"
output."jit-eval.err" = "2"

[envs.jit-lvn-eval]
command = "bril2json < {filename} | python3 ../jit.py {filename}.trace.err | python3 ../lvn.py | brili {args}"
output.out = "-"
output."jit-lvn-eval.err" = "2"

[envs.jit-lvn-tdce-eval]
command = "bril2json < {filename} | python3 ../jit.py {filename}.trace.err | python3 ../lvn.py | python3 ../tdce.py | brili {args}"
output.out = "-"
output."jit-lvn-tdce-eval.err" = "2"
