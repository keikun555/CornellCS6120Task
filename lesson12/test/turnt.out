1..61
ok 1 - benchmarks/core/ackermann.bril original-eval
ok 2 - benchmarks/core/ackermann.bril copy-eval
ok 3 - benchmarks/core/ackermann.bril trace
ok 4 - benchmarks/core/ackermann.bril jit
ok 5 - benchmarks/core/ackermann.bril jit-eval
ok 6 - benchmarks/core/ackermann.bril jit-lvn-eval
ok 7 - benchmarks/core/ackermann.bril jit-lvn-tdce-eval
ok 8 - benchmarks/core/armstrong.bril original-eval
ok 9 - benchmarks/core/armstrong.bril copy-eval
ok 10 - benchmarks/core/armstrong.bril trace
ok 11 - benchmarks/core/armstrong.bril jit
ok 12 - benchmarks/core/armstrong.bril jit-eval
ok 13 - benchmarks/core/armstrong.bril jit-lvn-eval
ok 14 - benchmarks/core/armstrong.bril jit-lvn-tdce-eval
ok 15 - benchmarks/core/binary-fmt.bril original-eval
ok 16 - benchmarks/core/binary-fmt.bril copy-eval
ok 17 - benchmarks/core/binary-fmt.bril trace
ok 18 - benchmarks/core/binary-fmt.bril jit
ok 19 - benchmarks/core/binary-fmt.bril jit-eval
ok 20 - benchmarks/core/binary-fmt.bril jit-lvn-eval
ok 21 - benchmarks/core/binary-fmt.bril jit-lvn-tdce-eval
ok 22 - benchmarks/core/bitshift.bril original-eval
ok 23 - benchmarks/core/bitshift.bril copy-eval
ok 24 - benchmarks/core/bitshift.bril trace
ok 25 - benchmarks/core/bitshift.bril jit
ok 26 - benchmarks/core/bitshift.bril jit-eval
ok 27 - benchmarks/core/bitshift.bril jit-lvn-eval
ok 28 - benchmarks/core/bitshift.bril jit-lvn-tdce-eval
ok 29 - benchmarks/core/bitwise-ops.bril original-eval
ok 30 - benchmarks/core/bitwise-ops.bril copy-eval
ok 31 - benchmarks/core/bitwise-ops.bril trace
ok 32 - benchmarks/core/bitwise-ops.bril jit
ok 33 - benchmarks/core/bitwise-ops.bril jit-eval
ok 34 - benchmarks/core/bitwise-ops.bril jit-lvn-eval
ok 35 - benchmarks/core/bitwise-ops.bril jit-lvn-tdce-eval
ok 36 - benchmarks/core/catalan.bril original-eval
ok 37 - benchmarks/core/catalan.bril copy-eval
ok 38 - benchmarks/core/catalan.bril trace
ok 39 - benchmarks/core/catalan.bril jit
ok 40 - benchmarks/core/catalan.bril jit-eval
ok 41 - benchmarks/core/catalan.bril jit-lvn-eval
ok 42 - benchmarks/core/catalan.bril jit-lvn-tdce-eval
ok 43 - benchmarks/core/check-primes.bril original-eval
ok 44 - benchmarks/core/check-primes.bril copy-eval
ok 45 - benchmarks/core/check-primes.bril trace
ok 46 - benchmarks/core/check-primes.bril jit
ok 47 - benchmarks/core/check-primes.bril jit-eval
ok 48 - benchmarks/core/check-primes.bril jit-lvn-eval
ok 49 - benchmarks/core/check-primes.bril jit-lvn-tdce-eval
ok 50 - benchmarks/core/collatz.bril original-eval
ok 51 - benchmarks/core/collatz.bril copy-eval
ok 52 - benchmarks/core/collatz.bril trace
ok 53 - benchmarks/core/collatz.bril jit
ok 54 - benchmarks/core/collatz.bril jit-eval
ok 55 - benchmarks/core/collatz.bril jit-lvn-eval
ok 56 - benchmarks/core/collatz.bril jit-lvn-tdce-eval
ok 57 - benchmarks/core/digital-root.bril original-eval
ok 58 - benchmarks/core/digital-root.bril copy-eval
ok 59 - benchmarks/core/digital-root.bril trace
ok 60 - benchmarks/core/digital-root.bril jit
ok 61 - benchmarks/core/digital-root.bril jit-eval
ok 62 - benchmarks/core/digital-root.bril jit-lvn-eval
ok 63 - benchmarks/core/digital-root.bril jit-lvn-tdce-eval
ok 64 - benchmarks/core/dot-product.bril original-eval
ok 65 - benchmarks/core/dot-product.bril copy-eval
ok 66 - benchmarks/core/dot-product.bril trace
ok 67 - benchmarks/core/dot-product.bril jit
ok 68 - benchmarks/core/dot-product.bril jit-eval
ok 69 - benchmarks/core/dot-product.bril jit-lvn-eval
ok 70 - benchmarks/core/dot-product.bril jit-lvn-tdce-eval
ok 71 - benchmarks/core/euclid.bril original-eval
ok 72 - benchmarks/core/euclid.bril copy-eval
ok 73 - benchmarks/core/euclid.bril trace
ok 74 - benchmarks/core/euclid.bril jit
ok 75 - benchmarks/core/euclid.bril jit-eval
ok 76 - benchmarks/core/euclid.bril jit-lvn-eval
ok 77 - benchmarks/core/euclid.bril jit-lvn-tdce-eval
ok 78 - benchmarks/core/fact.bril original-eval
ok 79 - benchmarks/core/fact.bril copy-eval
ok 80 - benchmarks/core/fact.bril trace
ok 81 - benchmarks/core/fact.bril jit
ok 82 - benchmarks/core/fact.bril jit-eval
ok 83 - benchmarks/core/fact.bril jit-lvn-eval
ok 84 - benchmarks/core/fact.bril jit-lvn-tdce-eval
ok 85 - benchmarks/core/factors.bril original-eval
ok 86 - benchmarks/core/factors.bril copy-eval
ok 87 - benchmarks/core/factors.bril trace
ok 88 - benchmarks/core/factors.bril jit
ok 89 - benchmarks/core/factors.bril jit-eval
ok 90 - benchmarks/core/factors.bril jit-lvn-eval
ok 91 - benchmarks/core/factors.bril jit-lvn-tdce-eval
ok 92 - benchmarks/core/fizz-buzz.bril original-eval
ok 93 - benchmarks/core/fizz-buzz.bril copy-eval
ok 94 - benchmarks/core/fizz-buzz.bril trace
ok 95 - benchmarks/core/fizz-buzz.bril jit
ok 96 - benchmarks/core/fizz-buzz.bril jit-eval
ok 97 - benchmarks/core/fizz-buzz.bril jit-lvn-eval
ok 98 - benchmarks/core/fizz-buzz.bril jit-lvn-tdce-eval
ok 99 - benchmarks/core/gcd.bril original-eval
ok 100 - benchmarks/core/gcd.bril copy-eval
ok 101 - benchmarks/core/gcd.bril trace
ok 102 - benchmarks/core/gcd.bril jit
ok 103 - benchmarks/core/gcd.bril jit-eval
ok 104 - benchmarks/core/gcd.bril jit-lvn-eval
ok 105 - benchmarks/core/gcd.bril jit-lvn-tdce-eval
ok 106 - benchmarks/core/hanoi.bril original-eval
ok 107 - benchmarks/core/hanoi.bril copy-eval
ok 108 - benchmarks/core/hanoi.bril trace
ok 109 - benchmarks/core/hanoi.bril jit
ok 110 - benchmarks/core/hanoi.bril jit-eval
ok 111 - benchmarks/core/hanoi.bril jit-lvn-eval
ok 112 - benchmarks/core/hanoi.bril jit-lvn-tdce-eval
ok 113 - benchmarks/core/loopfact.bril original-eval
ok 114 - benchmarks/core/loopfact.bril copy-eval
ok 115 - benchmarks/core/loopfact.bril trace
ok 116 - benchmarks/core/loopfact.bril jit
ok 117 - benchmarks/core/loopfact.bril jit-eval
ok 118 - benchmarks/core/loopfact.bril jit-lvn-eval
ok 119 - benchmarks/core/loopfact.bril jit-lvn-tdce-eval
ok 120 - benchmarks/core/mod_inv.bril original-eval
ok 121 - benchmarks/core/mod_inv.bril copy-eval
ok 122 - benchmarks/core/mod_inv.bril trace
ok 123 - benchmarks/core/mod_inv.bril jit
ok 124 - benchmarks/core/mod_inv.bril jit-eval
ok 125 - benchmarks/core/mod_inv.bril jit-lvn-eval
ok 126 - benchmarks/core/mod_inv.bril jit-lvn-tdce-eval
ok 127 - benchmarks/core/orders.bril original-eval
ok 128 - benchmarks/core/orders.bril copy-eval
ok 129 - benchmarks/core/orders.bril trace
ok 130 - benchmarks/core/orders.bril jit
ok 131 - benchmarks/core/orders.bril jit-eval
ok 132 - benchmarks/core/orders.bril jit-lvn-eval
ok 133 - benchmarks/core/orders.bril jit-lvn-tdce-eval
ok 134 - benchmarks/core/palindrome.bril original-eval
ok 135 - benchmarks/core/palindrome.bril copy-eval
ok 136 - benchmarks/core/palindrome.bril trace
ok 137 - benchmarks/core/palindrome.bril jit
ok 138 - benchmarks/core/palindrome.bril jit-eval
ok 139 - benchmarks/core/palindrome.bril jit-lvn-eval
ok 140 - benchmarks/core/palindrome.bril jit-lvn-tdce-eval
ok 141 - benchmarks/core/pascals-row.bril original-eval
ok 142 - benchmarks/core/pascals-row.bril copy-eval
ok 143 - benchmarks/core/pascals-row.bril trace
ok 144 - benchmarks/core/pascals-row.bril jit
ok 145 - benchmarks/core/pascals-row.bril jit-eval
ok 146 - benchmarks/core/pascals-row.bril jit-lvn-eval
ok 147 - benchmarks/core/pascals-row.bril jit-lvn-tdce-eval
ok 148 - benchmarks/core/perfect.bril original-eval
ok 149 - benchmarks/core/perfect.bril copy-eval
ok 150 - benchmarks/core/perfect.bril trace
ok 151 - benchmarks/core/perfect.bril jit
ok 152 - benchmarks/core/perfect.bril jit-eval
ok 153 - benchmarks/core/perfect.bril jit-lvn-eval
ok 154 - benchmarks/core/perfect.bril jit-lvn-tdce-eval
ok 155 - benchmarks/core/primes-between.bril original-eval
ok 156 - benchmarks/core/primes-between.bril copy-eval
ok 157 - benchmarks/core/primes-between.bril trace
ok 158 - benchmarks/core/primes-between.bril jit
ok 159 - benchmarks/core/primes-between.bril jit-eval
ok 160 - benchmarks/core/primes-between.bril jit-lvn-eval
ok 161 - benchmarks/core/primes-between.bril jit-lvn-tdce-eval
ok 162 - benchmarks/core/pythagorean_triple.bril original-eval
ok 163 - benchmarks/core/pythagorean_triple.bril copy-eval
ok 164 - benchmarks/core/pythagorean_triple.bril trace
ok 165 - benchmarks/core/pythagorean_triple.bril jit
ok 166 - benchmarks/core/pythagorean_triple.bril jit-eval
ok 167 - benchmarks/core/pythagorean_triple.bril jit-lvn-eval
ok 168 - benchmarks/core/pythagorean_triple.bril jit-lvn-tdce-eval
ok 169 - benchmarks/core/quadratic.bril original-eval
ok 170 - benchmarks/core/quadratic.bril copy-eval
ok 171 - benchmarks/core/quadratic.bril trace
ok 172 - benchmarks/core/quadratic.bril jit
ok 173 - benchmarks/core/quadratic.bril jit-eval
ok 174 - benchmarks/core/quadratic.bril jit-lvn-eval
ok 175 - benchmarks/core/quadratic.bril jit-lvn-tdce-eval
ok 176 - benchmarks/core/recfact.bril original-eval
ok 177 - benchmarks/core/recfact.bril copy-eval
ok 178 - benchmarks/core/recfact.bril trace
ok 179 - benchmarks/core/recfact.bril jit
ok 180 - benchmarks/core/recfact.bril jit-eval
ok 181 - benchmarks/core/recfact.bril jit-lvn-eval
ok 182 - benchmarks/core/recfact.bril jit-lvn-tdce-eval
ok 183 - benchmarks/core/rectangles-area-difference.bril original-eval
ok 184 - benchmarks/core/rectangles-area-difference.bril copy-eval
ok 185 - benchmarks/core/rectangles-area-difference.bril trace
ok 186 - benchmarks/core/rectangles-area-difference.bril jit
ok 187 - benchmarks/core/rectangles-area-difference.bril jit-eval
ok 188 - benchmarks/core/rectangles-area-difference.bril jit-lvn-eval
ok 189 - benchmarks/core/rectangles-area-difference.bril jit-lvn-tdce-eval
ok 190 - benchmarks/core/relative-primes.bril original-eval
ok 191 - benchmarks/core/relative-primes.bril copy-eval
ok 192 - benchmarks/core/relative-primes.bril trace
ok 193 - benchmarks/core/relative-primes.bril jit
ok 194 - benchmarks/core/relative-primes.bril jit-eval
ok 195 - benchmarks/core/relative-primes.bril jit-lvn-eval
ok 196 - benchmarks/core/relative-primes.bril jit-lvn-tdce-eval
ok 197 - benchmarks/core/reverse.bril original-eval
ok 198 - benchmarks/core/reverse.bril copy-eval
ok 199 - benchmarks/core/reverse.bril trace
ok 200 - benchmarks/core/reverse.bril jit
ok 201 - benchmarks/core/reverse.bril jit-eval
ok 202 - benchmarks/core/reverse.bril jit-lvn-eval
ok 203 - benchmarks/core/reverse.bril jit-lvn-tdce-eval
ok 204 - benchmarks/core/sum-bits.bril original-eval
ok 205 - benchmarks/core/sum-bits.bril copy-eval
ok 206 - benchmarks/core/sum-bits.bril trace
ok 207 - benchmarks/core/sum-bits.bril jit
ok 208 - benchmarks/core/sum-bits.bril jit-eval
ok 209 - benchmarks/core/sum-bits.bril jit-lvn-eval
ok 210 - benchmarks/core/sum-bits.bril jit-lvn-tdce-eval
ok 211 - benchmarks/core/sum-divisors.bril original-eval
ok 212 - benchmarks/core/sum-divisors.bril copy-eval
ok 213 - benchmarks/core/sum-divisors.bril trace
ok 214 - benchmarks/core/sum-divisors.bril jit
ok 215 - benchmarks/core/sum-divisors.bril jit-eval
ok 216 - benchmarks/core/sum-divisors.bril jit-lvn-eval
ok 217 - benchmarks/core/sum-divisors.bril jit-lvn-tdce-eval
ok 218 - benchmarks/core/sum-sq-diff.bril original-eval
ok 219 - benchmarks/core/sum-sq-diff.bril copy-eval
ok 220 - benchmarks/core/sum-sq-diff.bril trace
ok 221 - benchmarks/core/sum-sq-diff.bril jit
ok 222 - benchmarks/core/sum-sq-diff.bril jit-eval
ok 223 - benchmarks/core/sum-sq-diff.bril jit-lvn-eval
ok 224 - benchmarks/core/sum-sq-diff.bril jit-lvn-tdce-eval
ok 225 - benchmarks/core/totient.bril original-eval
ok 226 - benchmarks/core/totient.bril copy-eval
ok 227 - benchmarks/core/totient.bril trace
ok 228 - benchmarks/core/totient.bril jit
ok 229 - benchmarks/core/totient.bril jit-eval
ok 230 - benchmarks/core/totient.bril jit-lvn-eval
ok 231 - benchmarks/core/totient.bril jit-lvn-tdce-eval
ok 232 - benchmarks/core/up-arrow.bril original-eval
ok 233 - benchmarks/core/up-arrow.bril copy-eval
ok 234 - benchmarks/core/up-arrow.bril trace
ok 235 - benchmarks/core/up-arrow.bril jit
ok 236 - benchmarks/core/up-arrow.bril jit-eval
ok 237 - benchmarks/core/up-arrow.bril jit-lvn-eval
ok 238 - benchmarks/core/up-arrow.bril jit-lvn-tdce-eval
ok 239 - benchmarks/float/conjugate-gradient.bril original-eval
ok 240 - benchmarks/float/conjugate-gradient.bril copy-eval
ok 241 - benchmarks/float/conjugate-gradient.bril trace
ok 242 - benchmarks/float/conjugate-gradient.bril jit
ok 243 - benchmarks/float/conjugate-gradient.bril jit-eval
ok 244 - benchmarks/float/conjugate-gradient.bril jit-lvn-eval
ok 245 - benchmarks/float/conjugate-gradient.bril jit-lvn-tdce-eval
ok 246 - benchmarks/float/cordic.bril original-eval
ok 247 - benchmarks/float/cordic.bril copy-eval
ok 248 - benchmarks/float/cordic.bril trace
ok 249 - benchmarks/float/cordic.bril jit
ok 250 - benchmarks/float/cordic.bril jit-eval
ok 251 - benchmarks/float/cordic.bril jit-lvn-eval
ok 252 - benchmarks/float/cordic.bril jit-lvn-tdce-eval
ok 253 - benchmarks/float/euler.bril original-eval
ok 254 - benchmarks/float/euler.bril copy-eval
ok 255 - benchmarks/float/euler.bril trace
ok 256 - benchmarks/float/euler.bril jit
ok 257 - benchmarks/float/euler.bril jit-eval
ok 258 - benchmarks/float/euler.bril jit-lvn-eval
ok 259 - benchmarks/float/euler.bril jit-lvn-tdce-eval
ok 260 - benchmarks/float/mandelbrot.bril original-eval
ok 261 - benchmarks/float/mandelbrot.bril copy-eval
ok 262 - benchmarks/float/mandelbrot.bril trace
ok 263 - benchmarks/float/mandelbrot.bril jit
ok 264 - benchmarks/float/mandelbrot.bril jit-eval
ok 265 - benchmarks/float/mandelbrot.bril jit-lvn-eval
ok 266 - benchmarks/float/mandelbrot.bril jit-lvn-tdce-eval
ok 267 - benchmarks/float/newton.bril original-eval
ok 268 - benchmarks/float/newton.bril copy-eval
ok 269 - benchmarks/float/newton.bril trace
ok 270 - benchmarks/float/newton.bril jit
ok 271 - benchmarks/float/newton.bril jit-eval
ok 272 - benchmarks/float/newton.bril jit-lvn-eval
ok 273 - benchmarks/float/newton.bril jit-lvn-tdce-eval
ok 274 - benchmarks/float/norm.bril original-eval
ok 275 - benchmarks/float/norm.bril copy-eval
ok 276 - benchmarks/float/norm.bril trace
ok 277 - benchmarks/float/norm.bril jit
ok 278 - benchmarks/float/norm.bril jit-eval
ok 279 - benchmarks/float/norm.bril jit-lvn-eval
ok 280 - benchmarks/float/norm.bril jit-lvn-tdce-eval
ok 281 - benchmarks/float/n_root.bril original-eval
ok 282 - benchmarks/float/n_root.bril copy-eval
ok 283 - benchmarks/float/n_root.bril trace
ok 284 - benchmarks/float/n_root.bril jit
ok 285 - benchmarks/float/n_root.bril jit-eval
ok 286 - benchmarks/float/n_root.bril jit-lvn-eval
ok 287 - benchmarks/float/n_root.bril jit-lvn-tdce-eval
ok 288 - benchmarks/float/pow.bril original-eval
ok 289 - benchmarks/float/pow.bril copy-eval
ok 290 - benchmarks/float/pow.bril trace
ok 291 - benchmarks/float/pow.bril jit
ok 292 - benchmarks/float/pow.bril jit-eval
ok 293 - benchmarks/float/pow.bril jit-lvn-eval
ok 294 - benchmarks/float/pow.bril jit-lvn-tdce-eval
ok 295 - benchmarks/float/ray-sphere-intersection.bril original-eval
ok 296 - benchmarks/float/ray-sphere-intersection.bril copy-eval
ok 297 - benchmarks/float/ray-sphere-intersection.bril trace
ok 298 - benchmarks/float/ray-sphere-intersection.bril jit
ok 299 - benchmarks/float/ray-sphere-intersection.bril jit-eval
ok 300 - benchmarks/float/ray-sphere-intersection.bril jit-lvn-eval
ok 301 - benchmarks/float/ray-sphere-intersection.bril jit-lvn-tdce-eval
ok 302 - benchmarks/float/riemann.bril original-eval
ok 303 - benchmarks/float/riemann.bril copy-eval
ok 304 - benchmarks/float/riemann.bril trace
ok 305 - benchmarks/float/riemann.bril jit
ok 306 - benchmarks/float/riemann.bril jit-eval
ok 307 - benchmarks/float/riemann.bril jit-lvn-eval
ok 308 - benchmarks/float/riemann.bril jit-lvn-tdce-eval
ok 309 - benchmarks/float/sqrt.bril original-eval
ok 310 - benchmarks/float/sqrt.bril copy-eval
ok 311 - benchmarks/float/sqrt.bril trace
ok 312 - benchmarks/float/sqrt.bril jit
ok 313 - benchmarks/float/sqrt.bril jit-eval
ok 314 - benchmarks/float/sqrt.bril jit-lvn-eval
ok 315 - benchmarks/float/sqrt.bril jit-lvn-tdce-eval
ok 316 - benchmarks/long/function_call.bril original-eval
ok 317 - benchmarks/long/function_call.bril copy-eval
ok 318 - benchmarks/long/function_call.bril trace
ok 319 - benchmarks/long/function_call.bril jit
ok 320 - benchmarks/long/function_call.bril jit-eval
ok 321 - benchmarks/long/function_call.bril jit-lvn-eval
ok 322 - benchmarks/long/function_call.bril jit-lvn-tdce-eval
ok 323 - benchmarks/mem/adj2csr.bril original-eval
ok 324 - benchmarks/mem/adj2csr.bril copy-eval
ok 325 - benchmarks/mem/adj2csr.bril trace
ok 326 - benchmarks/mem/adj2csr.bril jit
ok 327 - benchmarks/mem/adj2csr.bril jit-eval
ok 328 - benchmarks/mem/adj2csr.bril jit-lvn-eval
ok 329 - benchmarks/mem/adj2csr.bril jit-lvn-tdce-eval
ok 330 - benchmarks/mem/adler32.bril original-eval
ok 331 - benchmarks/mem/adler32.bril copy-eval
ok 332 - benchmarks/mem/adler32.bril trace
ok 333 - benchmarks/mem/adler32.bril jit
ok 334 - benchmarks/mem/adler32.bril jit-eval
ok 335 - benchmarks/mem/adler32.bril jit-lvn-eval
ok 336 - benchmarks/mem/adler32.bril jit-lvn-tdce-eval
ok 337 - benchmarks/mem/binary-search.bril original-eval
ok 338 - benchmarks/mem/binary-search.bril copy-eval
ok 339 - benchmarks/mem/binary-search.bril trace
ok 340 - benchmarks/mem/binary-search.bril jit
ok 341 - benchmarks/mem/binary-search.bril jit-eval
ok 342 - benchmarks/mem/binary-search.bril jit-lvn-eval
ok 343 - benchmarks/mem/binary-search.bril jit-lvn-tdce-eval
ok 344 - benchmarks/mem/bubblesort.bril original-eval
ok 345 - benchmarks/mem/bubblesort.bril copy-eval
ok 346 - benchmarks/mem/bubblesort.bril trace
ok 347 - benchmarks/mem/bubblesort.bril jit
ok 348 - benchmarks/mem/bubblesort.bril jit-eval
ok 349 - benchmarks/mem/bubblesort.bril jit-lvn-eval
ok 350 - benchmarks/mem/bubblesort.bril jit-lvn-tdce-eval
ok 351 - benchmarks/mem/eight-queens.bril original-eval
ok 352 - benchmarks/mem/eight-queens.bril copy-eval
ok 353 - benchmarks/mem/eight-queens.bril trace
ok 354 - benchmarks/mem/eight-queens.bril jit
ok 355 - benchmarks/mem/eight-queens.bril jit-eval
ok 356 - benchmarks/mem/eight-queens.bril jit-lvn-eval
ok 357 - benchmarks/mem/eight-queens.bril jit-lvn-tdce-eval
ok 358 - benchmarks/mem/fib.bril original-eval
ok 359 - benchmarks/mem/fib.bril copy-eval
ok 360 - benchmarks/mem/fib.bril trace
ok 361 - benchmarks/mem/fib.bril jit
ok 362 - benchmarks/mem/fib.bril jit-eval
ok 363 - benchmarks/mem/fib.bril jit-lvn-eval
ok 364 - benchmarks/mem/fib.bril jit-lvn-tdce-eval
ok 365 - benchmarks/mem/mat-mul.bril original-eval
ok 366 - benchmarks/mem/mat-mul.bril copy-eval
ok 367 - benchmarks/mem/mat-mul.bril trace
ok 368 - benchmarks/mem/mat-mul.bril jit
ok 369 - benchmarks/mem/mat-mul.bril jit-eval
ok 370 - benchmarks/mem/mat-mul.bril jit-lvn-eval
ok 371 - benchmarks/mem/mat-mul.bril jit-lvn-tdce-eval
ok 372 - benchmarks/mem/max-subarray.bril original-eval
ok 373 - benchmarks/mem/max-subarray.bril copy-eval
ok 374 - benchmarks/mem/max-subarray.bril trace
ok 375 - benchmarks/mem/max-subarray.bril jit
ok 376 - benchmarks/mem/max-subarray.bril jit-eval
ok 377 - benchmarks/mem/max-subarray.bril jit-lvn-eval
ok 378 - benchmarks/mem/max-subarray.bril jit-lvn-tdce-eval
ok 379 - benchmarks/mem/primitive-root.bril original-eval
ok 380 - benchmarks/mem/primitive-root.bril copy-eval
ok 381 - benchmarks/mem/primitive-root.bril trace
ok 382 - benchmarks/mem/primitive-root.bril jit
ok 383 - benchmarks/mem/primitive-root.bril jit-eval
ok 384 - benchmarks/mem/primitive-root.bril jit-lvn-eval
ok 385 - benchmarks/mem/primitive-root.bril jit-lvn-tdce-eval
ok 386 - benchmarks/mem/quicksort.bril original-eval
ok 387 - benchmarks/mem/quicksort.bril copy-eval
ok 388 - benchmarks/mem/quicksort.bril trace
ok 389 - benchmarks/mem/quicksort.bril jit
ok 390 - benchmarks/mem/quicksort.bril jit-eval
ok 391 - benchmarks/mem/quicksort.bril jit-lvn-eval
ok 392 - benchmarks/mem/quicksort.bril jit-lvn-tdce-eval
ok 393 - benchmarks/mem/sieve.bril original-eval
ok 394 - benchmarks/mem/sieve.bril copy-eval
ok 395 - benchmarks/mem/sieve.bril trace
ok 396 - benchmarks/mem/sieve.bril jit
ok 397 - benchmarks/mem/sieve.bril jit-eval
ok 398 - benchmarks/mem/sieve.bril jit-lvn-eval
ok 399 - benchmarks/mem/sieve.bril jit-lvn-tdce-eval
ok 400 - benchmarks/mem/two-sum.bril original-eval
ok 401 - benchmarks/mem/two-sum.bril copy-eval
ok 402 - benchmarks/mem/two-sum.bril trace
ok 403 - benchmarks/mem/two-sum.bril jit
ok 404 - benchmarks/mem/two-sum.bril jit-eval
ok 405 - benchmarks/mem/two-sum.bril jit-lvn-eval
ok 406 - benchmarks/mem/two-sum.bril jit-lvn-tdce-eval
ok 407 - benchmarks/mem/vsmul.bril original-eval
ok 408 - benchmarks/mem/vsmul.bril copy-eval
ok 409 - benchmarks/mem/vsmul.bril trace
ok 410 - benchmarks/mem/vsmul.bril jit
ok 411 - benchmarks/mem/vsmul.bril jit-eval
ok 412 - benchmarks/mem/vsmul.bril jit-lvn-eval
ok 413 - benchmarks/mem/vsmul.bril jit-lvn-tdce-eval
ok 414 - benchmarks/mixed/cholesky.bril original-eval
ok 415 - benchmarks/mixed/cholesky.bril copy-eval
ok 416 - benchmarks/mixed/cholesky.bril trace
ok 417 - benchmarks/mixed/cholesky.bril jit
ok 418 - benchmarks/mixed/cholesky.bril jit-eval
ok 419 - benchmarks/mixed/cholesky.bril jit-lvn-eval
ok 420 - benchmarks/mixed/cholesky.bril jit-lvn-tdce-eval
ok 421 - benchmarks/mixed/mat-inv.bril original-eval
ok 422 - benchmarks/mixed/mat-inv.bril copy-eval
ok 423 - benchmarks/mixed/mat-inv.bril trace
ok 424 - benchmarks/mixed/mat-inv.bril jit
ok 425 - benchmarks/mixed/mat-inv.bril jit-eval
ok 426 - benchmarks/mixed/mat-inv.bril jit-lvn-eval
ok 427 - benchmarks/mixed/mat-inv.bril jit-lvn-tdce-eval
